src/main/java/duke/ui/Ui.java

```diff
@@ -0,0 +1,65 @@
+package duke.ui;
+import java.util.Scanner;
+
+public class Ui {
+    /**
+     * Scanner that will be used for scannign user's input
+     */
+    private Scanner scanner;
+
+    public Ui() {
+        this.scanner = new Scanner(System.in);
+    }
+
+    /**
+     * Prints welcome message when Duke is run.
+     */
+    public void showWelcome() {
```

Comment:
I like that you use verbs to name your methods! Perhaps your method name could be named printWelcome instead. This should align more with the description of your method.